package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@pnp/action-cli-login",
-  "version": "3.0.0",
+  "version": "3.0.1",
   "description": "GitHub Action to log in to a tenant using CLI for Microsoft 365",
   "license": "MIT",
   "main": "dist/index.js",
```

This will fix the `npm run package` script.

```suggestion
  "main": "lib/main.js",
```